lib/diagnostic.rb

```diff
@@ -114,4 +114,4 @@
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person = Hash.new { |hash, person| hash[person] = "Sorry, #{person[:first_name]} does not exists"}'
```

Comment:
you could set the default return value by `person.default = "#{person[:first_name]} is great"`